### PR TITLE
Changes for restore API

### PR DESCRIPTION
**Pull request by somalaya**
Related broker PR https://github.com/AzureAD/ad-accounts-for-android/pull/2849

### CODE REVIEW

File: common/src/main/java/com/microsoft/identity/common/adal/internal/AuthenticationConstants.java [hunk 1]
```diff
@@ -1561,7 +1581,9 @@ public enum API {
             PASSTHROUGH(PASSTHROUGH_PATH, null, null),
             READ_RESTRICTIONS_MANAGER(READ_RESTRICTIONS_MANAGER_PATH, null, null),
             GET_PREFERRED_AUTH_METHOD(BrokerContentProvider.GET_PREFERRED_AUTH_METHOD, null, null),
-            BROKER_INDIVIDUAL_LOGS_UPLOAD(BROKER_INDIVIDUAL_LOGS_UPLOAD_PATH, null, null);
+            BROKER_INDIVIDUAL_LOGS_UPLOAD(BROKER_INDIVIDUAL_LOGS_UPLOAD_PATH, null, null),
+            BROKER_BACKED_UP_GET_MSA_ACCOUNTS(BROKER_BACKED_UP_GET_MSA_ACCOUNTS_PATH, BROKER_VERSION_4, null),
```
Comment (rpdome) [inline common/src/main/java/com/microsoft/identity/common/adal/internal/AuthenticationConstants.java]: These should be BROKER_VERSION_5?

File: common/src/main/java/com/microsoft/identity/common/adal/internal/AuthenticationConstants.java [hunk 2]
```diff
@@ -1561,7 +1588,8 @@ public enum API {
             PASSTHROUGH(PASSTHROUGH_PATH, null, null),
             READ_RESTRICTIONS_MANAGER(READ_RESTRICTIONS_MANAGER_PATH, null, null),
             GET_PREFERRED_AUTH_METHOD(BrokerContentProvider.GET_PREFERRED_AUTH_METHOD, null, null),
-            BROKER_INDIVIDUAL_LOGS_UPLOAD(BROKER_INDIVIDUAL_LOGS_UPLOAD_PATH, null, null);
+            BROKER_INDIVIDUAL_LOGS_UPLOAD(BROKER_INDIVIDUAL_LOGS_UPLOAD_PATH, null, null),
+            BROKER_RESTORE_MSA_ACCOUNTS_WITH_TRANSFER_TOKENS(BROKER_RESTORE_MSA_ACCOUNTS_WITH_TRANSFER_TOKENS_PATH, BROKER_VERSION_4, null);
```
Comment (rpdome) [inline common/src/main/java/com/microsoft/identity/common/adal/internal/AuthenticationConstants.java]: BROKER_VERSION_5?

**Review by p3dr0rv — APPROVED** [review]
lgtm, do we doc this broker version changes in AuthLibrariesApiReview?

**Comment by somalaya** [comment]
> lgtm, do we doc this broker version changes in AuthLibrariesApiReview?

Sure

File: common/src/main/java/com/microsoft/identity/common/adal/internal/AuthenticationConstants.java (hunk 2 above)
Comment (somalaya) [reply common/src/main/java/com/microsoft/identity/common/adal/internal/AuthenticationConstants.java]: done

**Review by tanmaymanolkar1 — APPROVED** [review]

**Pull request merged**